gslb/gslbutils/gdputils.go

```diff
@@ -221,8 +244,24 @@ func (gf *GlobalFilter) AddToFilter(gdp *gdpv1alpha1.GlobalDeploymentPolicy) {
 	if len(gdp.Spec.MatchRules.NamespaceSelector.Label) == 1 {
 		gf.NSFilter = createNewNSFilter(gdp.Spec.MatchRules.NamespaceSelector.Label)
 	}
+
+	// Find out the list of clusters which need 3rd party vips syncing, assume SyncTypeVirtualServices
+	// to be defailt.
+	thirdPartyVips := make(map[string]struct{})
```

Comment:
why does the map contain an untyped struct?

Reply:
This was designed as a set, to check if the cluster required thirdPartyVips. However, with the new way of taking the third party vips, this is not required anymore, removed it.